libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -234,11 +233,21 @@ AP_GPS_UBLOX::_verify_rate(uint8_t msg_class, uint8_t msg_id, uint8_t rate) {
                 _cfg_needs_save = true;
             }
             break;
+        case MSG_PVT:
+            if(rate == RATE_PVT) {
+                _unconfigured_messages &= ~CONFIG_RATE_PVT;
+            } else {
+                _configure_message_rate(msg_class, msg_id, RATE_PVT);
+                _unconfigured_messages |= CONFIG_RATE_PVT;
+                _cfg_needs_save = true;
+            }
```

Comment:
> The other reason I don't want to flag it as true here, is that the Tx buffer might be true and we might never get the PVT enable message out, but we would start dropping the POSLLH/VELNED messages.

Ah, I though of that but didn't remember we would lose the driver instance. So I agree, wait for the actual message to set it to true. But that makes me have another question, now that you removed setting the rate for POSLLH/VELNED right at the beginning, couldn't we lose the driver instance too fast?